python/triton/ops/matmul.py

```diff
@@ -84,7 +84,8 @@ def _kernel(A, B, C, M, N, K,
             allow_tf32: tl.constexpr,
             fp8_fast_accum: tl.constexpr,
             BLOCK_M: tl.constexpr, BLOCK_N: tl.constexpr, BLOCK_K: tl.constexpr,
-            GROUP_M: tl.constexpr, SPLIT_K: tl.constexpr, EVEN_K: tl.constexpr, AB_DTYPE: tl.constexpr
+            GROUP_M: tl.constexpr, SPLIT_K: tl.constexpr, EVEN_K: tl.constexpr,
+            AB_DTYPE: tl.constexpr, C_DTYPE: tl.constexpr
```

Comment:
No need to pass `C_DTYPE`, since we already have it through `C.dtype.element_ty`.